Add pages for spring training events

xamberl opened the pull request:
Preview page (when it generates): https://chtc.github.io/web-preview/preview-spring-2025-training/events.html

ChristinaLK commented on _events/2025-01-14-information-session.md:
```diff
@@ -0,0 +1,68 @@
+CHTC staff will present an overview of CHTC services and how these services can help 
+researchers accomplish their research and computational goals. CHTC staff will also 
+help attendees identify the next steps for getting started - whether that is 
+getting an account, how to log in, or how to start running work. After 
```
```suggestion
getting an account, how to log in, or how to start running work. If you are attending in person, after 
```

ChristinaLK commented on _events/2025-02-11-containers.md:
```diff
@@ -0,0 +1,59 @@
+
+### Who
+
+* CHTC users who need to install software on our High Throughput or High Performance Computing system
```
@aowen-uwmad are you comfortable with both HPC + HTC?

ChristinaLK commented on _events/2025-02-11-containers.md:
```diff
@@ -0,0 +1,59 @@
+
+### When
+
+Tuesday, April 8, 10:30am - 12:00pm
```
```suggestion
Tuesday, February 11, 10:30am - 12:00pm
```

ChristinaLK reviewed (commented):
* For the workshops, do we want to add a question to the form that confirms people have an account? 
* Date of the March event is missing the leading 0 of the month in the file name. 
* It would be nice to have some picture variety between the workshops. I either search the campus photo library for generic photos or I go through my collection of nice "facilitator-y" images. Will share some links.

xamberl commented:
- Updated the registration forms for the February and April workshop to ask if they have CHTC accounts.
- Updated events with new images for some variety
- Fixed the name of the March event markdown file

ChristinaLK reviewed (approved):
Looks great!!! 😍
